### ISSUE TITLE
K8SSAND-792 ⁃ System-logger image running as root

### ISSUE THREAD

**Issue by RoachMJ** [k8ssandra/cass-operator#158]
**What happened?**
The system-logger image is running as a root user 

**Did you expect to see something different?**
As a best practice system-logger should be utilizing a user with least permissions for security concerns. 
**How to reproduce it (as minimally and precisely as possible)**:

**Environment**

* Cass Operator version:
* Cass-operator 1.7.1
k8ssandra/system-logger:9c4c3692
sha256:4dda0db106cb6543d0c7b7393f6164f03446dceb636d545be013b8e44f9cf4cc

    `Insert image tag or Git SHA here`
    
    * Kubernetes version information:    `kubectl version`
    * Kubernetes cluster kind:```
insert how you created your cluster: kops, bootkube, etc.
```

* Manifests:

```
insert manifests relevant to the issue
```

* Cass Operator Logs:

```
insert Cass Operator logs relevant to the issue here
```

**Anything else we need to know?**:



┆Issue is synchronized with this [Jira Bug](https://k8ssandra.atlassian.net/browse/K8SSAND-792) by [Unito](https://www.unito.io)
┆friendlyId: K8SSAND-792
┆priority: Medium

**Comment by burmanm** [comment]
That's true. This should probably be using the same user as k8ssandra/management-api-for-apache-cassandra we build.

Would you be interested in providing a PR?

**Comment by RoachMJ** [comment]
@burmanm Will do,  Thank you for the input.

**Comment by burmanm** [comment]
Fixed in v1.8.0
